.gitattributes

```diff
@@ -10,4 +10,5 @@
 Makefile text
 *.bat eol=crlf
 *.txt text
-*.sh text
+*.sh eol=lf
+Makefile eol=lf
```

Comment:
Makefile is referenced twice (text + eol) - it is technically correct, but possibly confusing